documentation/docs/scenarios/changeManagement.md

```diff
@@ -1,343 +1,92 @@
-                                                ]
-```
-
-## Upload a File to a Transport Request (`transportRequestUploadFile`)
-
-Upload a file to your transport request for a change document in SAP Solution Manager.
-
-### Prerequisites
-
-You have built your Java sources. For an example, see [mtaBuild](https://sap.github.io/jenkins-library/steps/mtaBuild/).
-
-### Mandatory Parameters
-
-| Parameter | Description |
-| --- | --- |
-| `script` | The common script envoronment of the running Jenkinsfile. The reference to the script that calls the pipeline step is privided by the `this` parameter, as in `script: this`. This allows the function to access the `commonPipelineEnvironment` to retrieve configuration parameters. |
-| `changeDocumentId` | The ID of the change document related to the transport request that is to be released. The ID is retrieved from the Git commit history. |
-| `transportRequestId` | The ID of the transport request to release. The ID is retreved from the Git commit history. |
-| `applicationId` | The ID of the application. |
-| `filePath` | The path of the file to upload. |
-| `changeManagement/credentialsId` | The ID of the credentials that are required to connect to SAP Solution Manager. |
-| `changeManagement/endpoint` | The address of SAP Solution Manager. |
-
-For an overview of the optional parameters, see **Parameters** in [transportRequestUploadFile](https://sap.github.io/jenkins-library/steps/transportRequestUploadFile/).
-
-### Step Configuration
-
-The step is configured by using a customer configuration file that is provided as a resource in a custom shared library.
-```
-@Library('piper-library-os@master') _
-
-// the shared lib containing the additional configuration
-// must be configured in Jenkins
-@Library(foo@master') __
-
-// in the shared lib denoted by 'foo', the additional configuration file
-// must be located under 'resources' ('resoures/myConfig.yml')
-prepareDefaultValues script: this, customDefaults: 'myConfig.yml'
-```
-Example content of `resources/myConfig.yaml` in branch `master` of the repository denoted by `foo`:
-```
-general:
-  changeManagement:
-    changeDocumentLabel: 'ChangeDocument\s?:'
-    cmClientOpts: '-Djavax.net.ssl.trustStore=<path to truststore>'
-    credentialsId: 'CM'
-    type: 'SOLMAN'
-    endpoint: 'https://example.org/cm'
-    git:
-      from: 'HEAD~1'
-      to: 'HEAD'
-      format: '%b'
-  ```
-  The properties configured in section `general/changeManagement` are shared between all steps related to change management.
-  You can also configure the properties on a per-step basis, for example:
-  ```
-  [...]
- steps:
-   transportRequestUploadFile:
-     applicationId: 'FOO'
-     changeManagement:
-       type: 'SOLMAN'
-       endpoint: 'https://example.org/cm'
-       [...]
+      endpoint: 'https://<backend-system>/sap/opu/odata/sap/AI_CRM_GW_CM_CI_SRV'
+      git:
+          from: 'HEAD~1'
```

Comment:
> from: 'HEAD~1'

Should not be set. Was only provided like this for the teched demo. The default is `master`. In essence we have to provide the base branch here. We make something like `master..HEAD` out if it, this means basically: scan all commits reachable from `HEAD`, but not reachable from `master`. I would like to omit this here totally.